pkg/server/server.go

```diff
@@ -648,7 +648,7 @@ func (s *Server) updateToken(ctx context.Context, db *sql.DB, uid string, endpoi
 		var err error
 		s.session, err = session.NewSession(
 			ctx,
-			fmt.Sprintf("https://%s/api/v1/session", endpoint),
+			fmt.Sprintf("https://%s", endpoint),
```

Comment:
Shall we make the https scheme as the part of endpoint?

Reply:
updated